nucypher/policy/conditions/time.py

```diff
@@ -0,0 +1,41 @@
+import time
+from marshmallow import fields, post_load
+from typing import Tuple
+
+from nucypher.policy.conditions._utils import CamelCaseSchema
+from nucypher.policy.conditions.base import ReencryptionCondition
+from nucypher.policy.conditions.lingo import ReturnValueTest
+
+
+class TimeCondition(ReencryptionCondition):
+    METHOD = 'timelock'
+
+    class Schema(CamelCaseSchema):
+        name = fields.Str()
+        method = fields.Str(default='timelock')
+        return_value_test = fields.Nested(ReturnValueTest.ReturnValueTestSchema())
+
+        @post_load
+        def make(self, data, **kwargs):
+            return TimeCondition(**data)
+
+    def __repr__(self) -> str:
+        r = f'{self.__class__.__name__}(timestamp={self.return_value_test.value})'
```

Comment:
Should the '=' in 'timestamp=' really be the comparator on the return_value_test? i.e.
```suggestion
        r = f'{self.__class__.__name__}(timestamp{self.return_value_test.comparator}{self.return_value_test.value})'
```